contents/docs/support-options.md

```diff
@@ -100,16 +104,15 @@ There are four ways to get free support:
 
 We are very grateful for folks that do this the other way around and help answer others' questions. You may just end up with some [merch](https://merch.posthog.com/) for particularly good answers :)
 
+## In-app Support
+
+Anybody using PostHog Cloud can raise a bug, submit feedback or otherwise ask for help via the help menu in the top right hand corner of the UI.  Your request will be routed to the correct team and we'll get back to you via email for further help if needed.
```

Comment:
@MarconLP do we want to set any kind of expectation here?

Reply:
We could list my focus goals here:
- high priority: 1 day response
- normal priority: 3 day response

But those would only apply to the product analytics and pipeline team tickets

Reply:
If we decide to put that somewhere the pricing page would probably be the better option.